autoload/lsp/ui/vim/code_action.vim

```diff
@@ -1,5 +1,7 @@
 " vint: -ProhibitUnusedVariable
 
+let s:popup_menu_is_available = exists('*popup_menu')
```

Comment:
we should move this check here. https://github.com/prabirshrestha/vim-lsp/blob/master/autoload/lsp/utils.vim. Old distros that has incomplete patches can be disabled in future if we see others reporting issues.